### DISCUSSION TITLE
반성합니다 ㅠㅠㅠㅠㅠ

### DISCUSSION THREAD

**Discussion by pykido** [BDD-CLUB/challenge-every-week-posting#562] [6. 반성문]
반성합니다 ㅠㅠㅠ.. 6월 들어서 배운 건 정말 많지만 정리한 적은 없는 거 같네요 ㅠㅠ... 옷만 잔뜩 사놓고 옷장 정리를 하나도 안한 거 같아 너무 기분이 찜찜합니다. 또 무조건 완벽한 블로그 글을 쓰고 싶다는 겉멋이 든 거 같습니다(?). 쨋든 이런저런 핑계들을 집어치우고 정리 또한 잘하는 개발자가 되도록 노력하겠습니다!!!

**Comment by stopmin** [comment]
옷부자 태윤
